samples/calendar/index.js

```diff
@@ -0,0 +1,143 @@
+
+
+const fs = require('fs');
+const readline = require('readline');
+const { google } = require('googleapis');
+
+// If modifying these scopes, delete token.json.
+const SCOPES = ['https://www.googleapis.com/auth/calendar.readonly'];
+// The file token.json stores the user's access and refresh tokens, and is
+// created automatically when the authorization flow completes for the first
+// time.
+const TOKEN_PATH = 'token.json';
+
+// Load client secrets from a local file.
+fs.readFile('credentials.json', (err, content) => {
+  if (err) { 
```

Comment:
A lot of this will be covered by making it consistent with the [drive sample](https://github.com/googleapis/google-api-nodejs-client/blob/master/samples/drive/list.js), but in the general sense we want samples to use `async/await` style syntax over callbacksies.